### PR TITLE
Uninstall updations

### PR DESCRIPTION
**Pull request by sinto-aot**
1. Updated uninstall.bat
2. Updated docker-compose.yml and sample.env
3. Updated docker-compose-arm64.yml 
https://aottech.atlassian.net/browse/FWF-2095

### CODE REVIEW

File: docker-compose/sample.env [hunk 1]
```diff
@@ -23,6 +23,7 @@
 ##URL of forms-flow-forms
 FORMIO_DEFAULT_PROJECT_URL=http://{your-ip-address}:3001
 #FORMIO_JWT_SECRET=--- change me now ---
+#NO_INSTALL=1
```
[the remark this answers is not in the document]
Comment (alanraju-aot) [reply docker-compose/sample.env:26]: default value should be 1 as we don't need the UI to be installed

**Review by alanraju-aot — APPROVED** [review]

File: scripts/uninstall.bat [hunk 2]
```diff
@@ -72,5 +72,7 @@ EXIT /B %ERRORLEVEL%
 
 :prune-docker
     docker volume prune -f
-    docker system prune
-    docker container prune
+    set images="31557345b2e8" "68141167a247" "8a7391477d59" "59baa6e3d1a8" "176399451347" "a12419875629" "360e5a526f8d"
```
Comment (sumesh-aot) [inline scripts/uninstall.bat:75]: Aren't these image ids local?

Comment (alanraju-aot) [reply scripts/uninstall.bat:75]: These images are the one's that is been downloaded from docker-hub to the local. This is just an alternate solution, we need to give updates for each release.

Comment (sreehari-aot) [reply scripts/uninstall.bat:75]: doesn't seem like a good idea to me.

Comment (sumesh-aot) [reply scripts/uninstall.bat:75]: Let's not use this. Have you tried using --filter with repository ? https://docs.docker.com/engine/reference/commandline/image_prune/#filter

Prune I believe is not a critical item, if we are worried about caching we can change the commands to use docker-compose pull.

**Review by sumesh-aot — COMMENTED** [review]
@sinto-aot good job, looks good on prune side, just a question why are we using 5.2.0-test image, shouldn't this be 5.2.0-alpha ?

**Comment by sinto-aot** [comment]
v5.2.0-alpha images are not labeled, so images are labeled in v5.2.0-test for testing

**Comment by alanraju-aot** [comment]
> @sinto-aot good job, looks good on prune side, just a question why are we using 5.2.0-test image, shouldn't this be 5.2.0-alpha ?

Yeah, the image should be 5.2.0-alpha. This was just created for testing purpose. Will generate the real image and also update the pipeline to create images using label.

**Review by alanraju-aot — APPROVED** [review]

**Review by abilpraju-aot — APPROVED** [review]

File: docker-compose/docker-compose-arm64.yml
Comment (alanraju-aot) [inline docker-compose/docker-compose-arm64.yml:1]: Remove the older env for client UI

File: scripts/uninstall.bat (hunk 2 above)
[the remark this answers is not in the document]
Comment (alanraju-aot) [reply scripts/uninstall.bat]: need to update for newer image versions

**Review by alanraju-aot — APPROVED** [review]
LGTM

**Review by alanraju-aot — APPROVED** [review]

**Pull request merged**